holoviews/core/data/interface.py

```diff
@@ -86,8 +86,12 @@ class Interface(param.Parameterized):
 
     datatype = None
 
+    # Denotes whether the interface expects gridded data
     gridded = False
 
+    # Denotes whether the interface expects multiple ragged arrays
```

Comment:
Does it have to be arrays? Isn't it ragged 'data' (i.e multiple elements of different lengths)?

Reply:
True.